frontend/src/modules/editor/global/editorDataLoader.js

```diff
@@ -107,11 +110,21 @@ define(function(require) {
     waitForLoad: function(callback) {
       var done = function() {
         if(preloader.hasLoadedData()) {
-          Origin.off('editor:dataPreloaded', done);
-          Origin.off('editor:dataLoaded', done);
+          Origin.off({
```

Comment:
Just noticed this; is there any reason for the duplicate code here and on lines 122–126? Could there just be a call to `removeEvents()`?